server/routes/templates.ts

```diff
@@ -589,7 +539,7 @@ function applicationBranchHandler(requestType: ApplicationType, anonymous: boole
 		}
 
 		if (!anonymous) {
-			let thisUser = await User.findById(user._id) as IUserMongoose;
+			let thisUser = await User.findById(user._id) as Model<IUser>;
 			// TODO this is a bug - dates are wrong
```

Comment:
Either remove the todo, make an issue, or fix this

Reply:
This TODO was introduced a while ago and is out of scope of this PR to fix

Reply:
Fair enough